resources/views/movies/edit.blade.php

```diff
@@ -0,0 +1,37 @@
+@extends('layout')
+
+@section('title', $movie->title)
+
+@section('content')
+    <div class="row">
+        <h5>Edit Movie</h5>
+        <form action="{{ route('movie.edit', ['movie' => $movie->id]) }}" method="post">
+            @csrf
+            <div class="form-group my-2">
+                <label for="title" class="form-label">{{ __('validation.attributes.title') }}</label>
+                <input id="title" type="text" placeholder="Title" name="title" value="{{ $movie->title }}"
```

Comment:
и в остальных полях тоже нужно поправить
```suggestion
                <input id="title" type="text" placeholder="Title" name="title" value="{{ old('title', $movie->title) }}"
```